dynawo/sources/Models/Modelica/Dynawo/Electrical/Transformers/TransformerFixedRatio.mo

```diff
@@ -12,7 +12,7 @@ within Dynawo.Electrical.Transformers;
 * This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
 */
 
-model TransformerFixedRatio "Two winding transformer with a fixed ratio"
+model TransformerFixedRatio "Two windings transformer with a fixed ratio"
```

Comment:
I always have doubts here: I think the right way to write this is Two-winding transformer, like a Three-year-old boy, can you maybe check with someone you trust on this?:)

Reply:
No you are right, it is two-winding transformer. My mistake ;)